Library/Tracking/KSRAnalytics.swift

```diff
@@ -657,14 +678,25 @@ public final class KSRAnalytics {
 
    - parameter sort: The new sort that was selected.
    */
-  public func trackDiscoverySelectedSort(nextSort sort: DiscoveryParams.Sort, params: DiscoveryParams) {
+  public func trackDiscoverySelectedSort(
+    prevSort sort: DiscoveryParams.Sort,
+    nextSort: DiscoveryParams.Sort,
+    params: DiscoveryParams
+  ) {
     let props = discoveryProperties(from: params)
       .withAllValuesFrom([
-        "discover_sort": sort.rawValue
+        "discover_sort": sort.trackString
```

Comment:
Yes it is, the requirement, from the analytics team is different from what exist in `discoveryProperties `

Reply:
i know. i'm saying that i see `sort` being assigned in `discoveryProperties` and I would like you to verify that this line and assignment here is necessary instead of within the method.

Reply:
I understand, the one defined in `discoveryProperties ` get's it's value from `DiscoveryParams`, but base on the ticket, the value is wrong, because the ticket expects `discover_sort` to be the previously selected item.